src/plugins/wizard/public/application/utils/wizard_state_validation.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Ajv from 'ajv';
+import wizardStateSchema from './schema.json';
+
+export const validateWizardState = (wizardState) => {
```

Comment:
This was my miss from the first review, but can we return a valid state explicitly? e.g.

```ts
(state) => { valid: boolean, errors?: Error }
```

Reply:
change the return to explicitly return a valid and error fields